android-components/components/feature/customtabs/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <color name="fx_mobile_private_text_color_primary">@color/photonLightGrey05</color>
```

Comment:
this is a bit of a flag for me. We shouldn't be using firefox colors inside of AC. See the next comment for a suggested approach